core/storage/email/gae_models_test.py

```diff
@@ -128,9 +149,101 @@ def test_get_by_hash_behavior_with_sent_datetime_lower_bound(self):
                     sent_datetime_lower_bound='Not a datetime object')
 
 
+class ReplyToIdModelUnitTests(test_utils.GenericTestBase):
+    """Test the GeneralFeedbackEmailReplyToIdModel class."""
+
+    def setUp(self):
+        super(ReplyToIdModelUnitTests, self).setUp()
+
+        def _generate_unique_id_for_tests(unused_cls):
+            return 'Non_unique_id'
+
+        self.generate_constant_id_ctx = self.swap(
+            email_models.GeneralFeedbackEmailReplyToIdModel,
+            '_generate_unique_reply_to_id',
+            types.MethodType(
+                _generate_unique_id_for_tests,
+                email_models.GeneralFeedbackEmailReplyToIdModel))
+
+    def test_id_generation_works_correctly(self):
```

Comment:
In what scenarios should this test fail? It seems to me like it might be testing an implementation detail (how IDs are formatted), and it's not clear to me at the moment whether we'd want situations like deciding to change how IDs are formatted to result in this test failing. What are your thoughts?

Reply:
Since I "fixed" the file I'm testing earlier this month because changes where made to the return type and the documentation was not changed accordingly, I think that kind of test should at least ensure that changes to format are thought about, but I agree it's a detail. I changed the name accordingly, and can understand why you would want to remove it